control/autoware_control_toolbox/version.txt

```diff
@@ -0,0 +1 @@
+v0.0.1
```

Comment:
I have not seen this file in other packages. Is it necessary ?
I think the version of ROS packages is supposed to be maintained in the `package.xml` file.

Reply:
Sorry, it was for my reference; I remove it.